stable/traefik/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.41.1
-appVersion: 1.6.5
+version: 1.42.0
```

Comment:
I know it's too late anyway, but should not this have been `1.41.2`? 🤔

Reply:
Arguably yes, but due to the severity, we bumped the minor to encourage users to update.